ts/screens/authentication/LandingScreen.tsx

```diff
@@ -120,15 +136,86 @@ class LandingScreen extends React.PureComponent<Props> {
     }
   }
 
+  private renderAndroidConditions = () => {
```

Comment:
Could it be a component included in its own file?
We should pass to it `hasCieApiLevelSupport`

Reply:
We may create a component for the complete child of the Modal including this block maybe?
```javascript
          <React.Fragment>
            <Markdown>
              {I18n.t("authentication.landing.cie_unsupported.body")}
            </Markdown>
            {Platform.OS === "android" && this.renderAndroidConditions()}
          </React.Fragment>
```

we should pass `hasCieApiLevelSupport` and `hasCieNFCFeature` too i guess

Reply:
@Undermaken 
The component is available in #05798b8